### PR TITLE
Add subscription calendar to recurring event

### PR DESCRIPTION
**Pull request by jancborchardt**
Please review @evalica @simonv3

### CODE REVIEW

File: osd-recurring-online-meeting.md [hunk 1]
```diff
@@ -14,7 +14,12 @@ Join us for our monthly get together where we talk about all things Open Source
 
 To see what we talked about in previous meetings, look at:
 
-[https://github.com/opensourcedesign/events/issues/49](https://github.com/opensourcedesign/events/issues/49)
+[github.com/opensourcedesign/events/issues/49](https://github.com/opensourcedesign/events/issues/49)
+
+You can also subscribe to the calendar with this meeting in it as well as other related Open Source Design events:
```
Comment (evalica) [inline osd-recurring-online-meeting.md:19]: I would remove the "with this meeting in it" text.

File: osd-recurring-online-meeting.md [hunk 2]
```diff
@@ -14,7 +14,12 @@ Join us for our monthly get together where we talk about all things Open Source
...
+
+[cloud.nextcloud.com/apps/calendar/public/MIFAFLFJADIVX63I](https://cloud.nextcloud.com/apps/calendar/public/MIFAFLFJADIVX63I)
```
Comment (evalica) [inline osd-recurring-online-meeting.md:21]: This also works: https://calendar.google.com/calendar/embed?src=f0lbq0gf24grn886l41qhnq0oq8usbse%40import.calendar.google.com

Comment (evalica) [reply osd-recurring-online-meeting.md:21]: Looks good in general.

**Review by evalica — APPROVED** [review]

File: osd-recurring-online-meeting.md (hunk 2 above)
Comment (jancborchardt) [reply osd-recurring-online-meeting.md:21]: Well, you have to give your email or phone number to Google there, so I’d like to avoid that. ;)

**Pull request merged**